src/components/default-page-template.less

```diff
@@ -47,13 +47,6 @@
     justify-content: flex-start;
   }
 
-  &.DefaultPageTemplate-headerContent--center {
-    flex: none;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-  }
-
```

Comment:
Now that the center is gone, we can give the left side more room (now it has max the same as the right side, but the right side only has two buttons)

Reply:
I've checked this and I feel it's ok, now they each take half, growing to however much each needs when the room shrinks:

<img width="281" alt="Screen Shot 2022-02-01 at 10 42 47" src="https://user-images.githubusercontent.com/8189923/151945686-07276243-2df6-4d30-be2d-ee059ae19284.png">

Or did I get this wrong? 🤔

Reply:
The left can never take more than 50% if both have a flex grow of 1, in case of OMA that gave me a headache on mobile as the name is very long and the left side would need much more space than the right.

Reply:
Simplified as discussed 👍